vios/docker/Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM ubuntu:20.04
```

Comment:
thanks @jaro0149 
would you know if using a more recent image would work with vios?
we tend to use debian:12. you can check SR OS image for reference

Reply:
thank you for code review

yes, it works - I have updated Dockerfile definition, I had to additionally install qemu-utils to make it work
appending docker logs
[clab-vios-lab-xrv1.log](https://github.com/user-attachments/files/17175714/clab-vios-lab-xrv1.log)